src/credentials/SupportedCredentialsConfiguration/PIDSupportedCredentialJptVCDM.ts

```diff
@@ -0,0 +1,259 @@
+import { CategorizedRawCredentialView, CategorizedRawCredentialViewRow } from "../../openid4vci/Metadata";
+import { VerifiableCredentialFormat } from "wallet-common/dist/types";
+import { VCDMSupportedCredentialProtocol } from "../../lib/CredentialIssuerConfig/SupportedCredentialProtocol";
+import { formatDateDDMMYYYY } from "../../lib/formatDate";
+import { urlToDataUrl } from "../../lib/urlToDataUrl";
+import { AuthorizationServerState } from "../../entities/AuthorizationServerState.entity";
+import { CredentialView } from "../../authorization/types";
+import { CredentialSigner } from "../../services/interfaces";
+import { JWK } from "jose";
+import base64url from "base64url";
+import { Request } from "express";
+import { issuerSigner } from "../../configuration/issuerSigner";
+import { parsePidData } from "../datasetParser";
+import path from "node:path";
+import fs from 'fs';
+import { AuthenticationChain, AuthenticationChainBuilder } from "../../authentication/AuthenticationComponent";
+import { CONSENT_ENTRYPOINT } from "../../authorization/constants";
+import { GenericLocalAuthenticationComponent } from "../../authentication/authenticationComponentTemplates/GenericLocalAuthenticationComponent";
+import { initializeCredentialEngine } from "../../lib/initializeCredentialEngine";
+import { pidMetadata1_8 } from "./typeMetadata/pidMetadata";
+import { pidSchema_1_8 } from "./schema/pidSchema";
+import { convertSdjwtvcToOpenid4vciClaims } from "../../lib/convertSdjwtvcToOpenid4vciClaims";
+import { createSRI } from "../../lib/sriGenerator";
+
+
+const datasetName = "vid-dataset.xlsx";
+parsePidData(path.join(__dirname, `../../../../dataset/${datasetName}`)) // test parse
+
+
+const splitBbsPidMetadata = pidMetadata1_8;
+
+export class PIDSupportedCredentialJptVCDM implements VCDMSupportedCredentialProtocol {
+	constructor() {}
+
+	getAuthenticationChain(): AuthenticationChain {
+		return new AuthenticationChainBuilder()
+			.addAuthenticationComponent(new GenericLocalAuthenticationComponent(this.getScope() + "-1-local", CONSENT_ENTRYPOINT, {
+				"pid_id": { datasetColumnName: "pid_id", parser: (val: any) => String(val) },
+			},
+				async () => parsePidData(path.join(__dirname, "../../../../dataset/" + datasetName)) as any[],
+				[{ username: "john", password: "secret" }, { username: "emily", password: "secret" }]
+			))
+			// .addAuthenticationComponent(new LocalAuthenticationComponent2("2-local", CONSENT_ENTRYPOINT))
+			.build();
+	}
+
+	getScope(): string {
+		return "pid:jpt_dc";
+	}
+
+	getCredentialSigner(): CredentialSigner {
+		return issuerSigner;
+	}
+
+	getId(): string {
+		return "urn:eudi:pid:1:dc:jpt";
+	}
+	getFormat(): VerifiableCredentialFormat {
+		return VerifiableCredentialFormat.DC_JPT;
+	}
+	getTypes(): string[] {
+		return ["VerifiableCredential", "VerifiableAttestation", "PID", this.getId()];
+	}
+
+	getDisplay() {
+		return {
+			name: `PID ARF 1.8 (${this.getFormat()})`,
```

Comment:
```suggestion
			name: `PID (${this.getFormat()})`,
```
just because we removed ARF references from other credentials